addons/website/static/src/js/editor/editor_menu.js

```diff
@@ -136,12 +134,14 @@ var EditorMenu = Widget.extend({
         return $wrapwrap.find('[data-oe-model]')
             .not('.o_not_editable')
             .filter(function () {
-                return !$(this).closest('.o_not_editable').length;
+                var $parent = $(this).closest('.o_editable, .o_not_editable');
+                return !$parent.length || $parent.hasClass('o_editable');
             })
             .not('link, script')
             .not('[data-oe-readonly]')
             .not('img[data-oe-field="arch"], br[data-oe-field="arch"], input[data-oe-field="arch"]')
             .not('.oe_snippet_editor')
+            .not('hr, br, input, textarea')
```

Comment:
add image ? to check please

Reply:
I check, it's good. https://github.com/odoo/odoo/pull/31183/commits/f857610c433b2b95168155c4cd804700b40faad5#diff-788a9b71279ba1f6a0dc4534569e9dc4R201